api/src/main/java/org/eclipse/microprofile/jwt/builder/spi/JwtProvider.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright (c) 2016-2017 Contributors to the Eclipse Foundation
+ *
+ *  See the NOTICE file(s) distributed with this work for additional
+ *  information regarding copyright ownership.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  You may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package org.eclipse.microprofile.jwt.builder.spi;
+
+import java.util.Iterator;
+import java.util.Map;
+import java.util.ServiceLoader;
+
+import org.eclipse.microprofile.jwt.builder.JwtClaimsBuilder;
+import org.eclipse.microprofile.jwt.builder.JwtException;
+
+/**
+ * Service provider for JWT Claims Builder objects.
+ *
+ * @see ServiceLoader
+ */
+public abstract class JwtProvider {
+    /**
+     * Name of the default {@code JwtProvider} implementation class.
+     */
+    private static final String DEFAULT_JWT_PROVIDER = "io.smallrye.jwt.build.impl.JwtProviderImpl";
```

Comment:
smallrye

Reply:
Will follow the resolution as per the above comment